.github/workflows/ci-notebooks.yml

```diff
@@ -0,0 +1,29 @@
+name: ci-notebooks
+on:
+  pull_request:
+    paths:
+      - modin/**
+      - examples/**
+jobs:
+  test-notebooks:
+    name: test notebooks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8.x"
+          architecture: "x64"
+      - name: Cache datasets
+        uses: actions/cache@v2
+        with:
+          path: taxi.csv
+          # update cache only if notebooks need it are changed
+          key: hashFiles("examples/tutorial/tutorial_notebooks/introduction/**")
+      - run: pip install pytest pytest-cov tqdm black flake8 flake8-print
```

Comment:
This file contains `modin[all]` requirement, so if we use requirements file, pip installable version of modin will be used, not the current version of repository. I would suggest to build modin instead of install it to keep notebooks up to date with repository code.

Reply:
That probably makes sense. We can install Modin from sources in the req.txt file (even with a concrete engine).

Reply:
Since we want to divide tutorials into sub-directories (pandas_on_ray/dask, omnisci) I think modin[ray] will be enough.

Reply:
Ok, added deps installation from requirements file (specifying relative path of repository root in requirements file can be tricky, so file is modified inplace to specify correct path). `.[all]` is used since notebooks for each sub-directory will also need `.[spreadsheet]` installation except target engine deps.

Reply:
I have been thinking on the source of installation and I tend to choose installation of stable releases after all. Also, we are not be able to install Modin on OmniSci from sources.

Not sure if spreadsheet works well with all the engines, even with OmniSci. @devin-petersohn, is it supposed to be worked?

Reply:
@devin-petersohn, what do you think, should we use modin installed from sources or installed via pip for testing notebooks?